src/test/java/http_server/ResponseMakerTest.java

```diff
@@ -0,0 +1,126 @@
+package http_server;
+
+import org.junit.Test;
+
+import static http_server.StatusCodes.*;
+import static org.junit.Assert.assertEquals;
+
+public class ResponseMakerTest {
+
+    private String responseStart = "HTTP/1.1 ";
+    private String space = " ";
+
+    private String buildResponse(String responseStart, String statusCode, String space, String statusMessage) {
+        return responseStart + statusCode + space + statusMessage;
```

Comment:
Do you need to pass `responseStart` in here every time you call it?  It looks like it never changes